src/charm.py

```diff
@@ -2,22 +2,200 @@
 # Copyright 2023 Canonical Ltd.
 # See LICENSE file for licensing details.
 
-"""Charmed operator for the 5G NSSF service."""
+"""Charmed operator for the SDCORE NSSF service."""
```

Comment:
```suggestion
"""Charmed operator for the SD-Core NSSF service."""
```

Reply:
Updated the workflows, and the `SD-Core` naming.